addon/mixins/popup.js

```diff
@@ -63,9 +63,11 @@ export default Mixin.create({
   didCreateLayer() {
     this._super(...arguments);
     if (this.get('hasBlock')) {
-      this._popup = this.L.popup({}, this._layer);
+
+      let popupOptions = this.get('popupOptions');
```

Comment:
since we're not reusing the value, just pass `this.get('popupOptions')` directly in the popup creation. Just to be aligned with the rest of the project's code style.